markdown/boomi-v2-getting-started/boomi-v2-getting-started.md

```diff
@@ -161,54 +158,37 @@ Select your newly created environment and search for your atom to attach it
 Positive
 : Note: a blue icon next to your newly created Atom means that it has an Online Status
 
-#### 5. Install the Solace Connector
-
-Click on the Settings > Account information for your account name
-![account_setup](img/account_setup.png "account_setup")
-
-In the left hand column, click on Publisher
-![publisher](img/publisher.png "publisher")
-Fill out the required information, making sure to click on the box for "Enable connector SDK development" Click on Save.
-
-Now, click on Developer at the bottom of the left hand column.
-![developer](img/developer.png "developer")
+#### 🎉 Viola! Now you have your Solace and Boomi environments setup
 
-Click on Add Connector Group and name it Solace
+## Check out your Event-Driven Architecture
```

Comment:
Line 144
Broken Link: [Environment Management Boomi User Guide](https://help.boomi.com/bundle/integration/page/c-atm-Environment_management_1ec94aeb-ffaf-4cec-a3b0-483c2af3967c.html)

Reply:
Good catch - I had this since v1 release, I just updated with a new link that is not broken